src/coreclr/gc/gc.cpp

```diff
@@ -33301,7 +33520,7 @@ void gc_heap::plan_phase (int condemned_gen_number)
     plan_generation_starts (consing_gen);
 #endif //!USE_REGIONS
 
-    descr_generations ("AP");
+    //descr_generations ("AP");
```

Comment:
Since this is dead code, can this be removed?

Reply:
it's not deadcode. I just commented it out to make the output less verbose. since I reverted dprintf changes in `descr_generations` I can revert this.